src/HotChocolate/Core/src/Abstractions/WellKnownContextData.cs

```diff
@@ -197,4 +197,14 @@ public static class WellKnownContextData
     /// The key to get the IdValue object from the context data.
     /// </summary>
     public const string IdValue = "HotChocolate.Relay.Node.Id.Value";
+
+    /// <summary>
+    /// The key to get check if a field is the node field.
+    /// </summary>
+    public const string IsNodeField = "HotChocolate.Relay.Node.IsNodeField";
+
+    /// <summary>
+    /// The key to get check if a field is the nodes field.
+    /// </summary>
+    public const string IsNodesField = "HotChocolate.Relay.Node.IsNodeField";
```

Comment:
Just came across this, while fixing a merge conflict in this file: Is it intentional that the value of both these context keys is the same or should this one be the plural variant? (Haven't reviewed the actual PR)

Reply:
good catch! thanks tobias!